neqo-transport/src/connection.rs

```diff
@@ -1494,6 +1507,8 @@ impl Connection {
                     "Received DataBlocked with data limit {}",
                     data_limit
                 );
+                // But if it does, open it up all the way
+                self.flow_mgr.borrow_mut().max_data(0x3FFF_FFFF_FFFF_FFFE);
```

Comment:
Should we make a constant for this?

BTW, did you intentionally send 2^62-2 as opposed to 2^62-1?

Reply:
This is just temp code that will be removed.

Reply:
On second thought, keep it but make the changes you suggested.